### ISSUE TITLE
RPC spec

### ISSUE THREAD

**Issue by not7cd** [farhi/matlab-starbook#1]
Hi, I am interested if you are in possession of a RPC documentation of starbook interface. I got my hands on ver 1.2, and trying to make it work with ver 2.7.

**Comment by farhi** [comment]
Hello,
I got the documentation from an email with Vixen Japan. They sent me what is reported at:

- http://farhi-rivasseau.blogspot.com/2010/07/telescope-perl-vixen-distant-ethernet.html

I also extracted more stuff from:

- https://github.com/rbur004/rubytelescopeserver

especially what relates to the screen shot.
Last, I searched for server commands in the Starbook binary code, but found none new. So I guess v1.2 is what is currenly used. The newest Starbook have more commands, but I do not own such mount.
